.rubocop_todo.yml

```diff
@@ -1,16 +1,32 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2017-10-11 13:16:21 -0400 using RuboCop version 0.50.0.
+# on 2018-10-25 13:54:35 -0400 using RuboCop version 0.59.2.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
```

Comment:
Just want to call out that this file looks like it's meant to get smaller but seems in fact to be getting bigger. If we care about rubocop linting ought we consider taking a moment to clean this up?

Reply:
Sure, but in a separate PR. My workflow for this is `rubocop -a ; rubocop --auto-gen-config`.